proto/p4/p4runtime.proto

```diff
@@ -356,6 +357,29 @@ message CloneSessionEntry {
   uint64 packet_length_bytes = 4;
 }
 
+//------------------------------------------------------------------------------
+// For writing and reading matches in a parser value set. A state transition
+// on an empty value set will never be taken. Matches contained in a value set
+// must be unique. The number of matches must be at most the size of the value
```

Comment:
"must be at most the size of the value set"
nit: Its a little vague where 'size' is coming from ... a reader may think 'size' is part of P4Runtime (e.g. message below). Consider appending "specified in the P4 program".

Reply:
Done